website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -35,12 +35,16 @@ When using defer, it compares artifacts from the most recent successful producti
 
 ### Defer in the dbt Cloud IDE
 
+For deferral in the IDE, you must have production artifacts available generated by a deploy job. dbt Cloud will first check for these artifacts in your Staging environment (if it exists) or, if not, the Production environment. 
+
+The defer feature in the IDE is not expected to work if a Staging environment exists but no deploy job has run there. The necessary metadata to power the defer feature will not exist until a deploy job has run successfully in the Staging environment.
+
 To enable defer in the dbt Cloud IDE, toggle the **Defer to production** button on the command bar. Once enabled, dbt Cloud will:
 
-1. Pull down the most recent manifest from the Production environment for comparison
+1. Pull down the most recent manifest from the Staging or Production environment for comparison
 2. Pass the `--defer` flag to the command (for any command that accepts the flag)
 
-For example, if you were to start developing on a new branch with [nothing in your development schema](/reference/node-selection/defer#usage), edit a single model, and run `dbt build -s state:modified` &mdash;  only the edited model would run. Any `{{ ref() }}` functions will point to the production location of the referenced models.
+If you were to start developing on a new branch with [nothing in your development schema](/reference/node-selection/defer#usage), edit a single model, and run `dbt build -s state:modified` &mdash;  only the edited model would run. Any `{{ ref() }}` functions will point to the staging or production location of the referenced models.
```

Comment:
sugg leaving the 'for example as mentioned [here](https://github.com/dbt-labs/docs.getdbt.com/pull/6856#issuecomment-2630569382) since this is an example scenario and we should let the users know that.
```suggestion
For example, if you were to start developing on a new branch with [nothing in your development schema](/reference/node-selection/defer#usage), edit a single model, and run `dbt build -s state:modified` &mdash;  only the edited model would run. Any `{{ ref() }}` functions will point to the staging or production location of the referenced models.
```